RxCocoa/iOS/UITabBar+Rx.swift

```diff
@@ -56,6 +56,17 @@ extension UITabBar {
     }
     
     /**
+     Bindable sink for `items` property.
+    
+     - parameter animated: A Boolean indicating whether changes should be animated.
+    */
+    public func rx_items(animated: Bool) -> AnyObserver<[UITabBarItem]?> {
```

Comment:
I'm wondering what others think about this signature.  I don't think it would be possible to pass this function around, since it overlaps with the `var rx_items`, however, I don't know if that's necessary or even helpful.

I tried to find as similar of a signature already in the codebase as possible, which was: https://github.com/ReactiveX/RxSwift/blob/2aebe14/RxCocoa/OSX/NSImageView+Rx.swift#L29